### DISCUSSION TITLE
项目路由

### DISCUSSION THREAD

**Discussion by aslrz-club** [empjs/emp#319] [General]
能提供一个vue2基站应用和子应用路由配置实战吗，感觉emp这个更像是提供一个公共的组件路由，没有那种基站加几个子应用聚合起来的那种吗

**Comment by MAXLZ1** [comment]
把子应用当成子组件就行了，基站的路由该怎么配就怎么配，根据路由显示对应子组件就行

**Comment by vellsuan** [comment]
这个比你说的先进！怎么还想要倒退的呢！

**Comment by ckken** [comment]
可以细化讨论下
